### PR TITLE
Ensure frames are correctly ordered in exported movies

### LINKED ISSUE #8: Exported video frame order may be scrambled
The frames are stored as files and then processed into a video, but the file read is not guaranteed to use any particular order.

### PR DESCRIPTION
**Pull request by dabeshouse**
Closes #8

### CODE REVIEW

File: src/main/java/movieMaker/MovieWriter.java [hunk 1]
```diff
@@ -220,8 +218,8 @@ public List getFrameNames()
   public boolean writeAVI()
   {
     JpegImagesToMovie imageToMovie = new JpegImagesToMovie();
-    List frameNames = getFrameNames();
-    Picture p = new Picture((String) frameNames.get(0));
+    List<String> frameNames = getFrameNames();
```
Comment (jennej) [inline src/main/java/movieMaker/MovieWriter.java:221]: what happens if frameNames is empty?

Comment (dabeshouse) [reply src/main/java/movieMaker/MovieWriter.java:221]: Nothing good.
Not sure how a user could cause it to happen easily, but I added a check.

Comment (jennej) [reply src/main/java/movieMaker/MovieWriter.java:221]: looks like you only got one of the two...

Comment (dabeshouse) [reply src/main/java/movieMaker/MovieWriter.java:221]: What!? Two places!?
You want me to look at all thirty changed lines and figure that out?

(Oops. Thanks for catching that.)

Fixed.

**Review by jennej — APPROVED** [review]

**Pull request merged**